src/pocketmine/tile/Chest.php

```diff
@@ -74,7 +74,7 @@ public function close(){
 	}
 
 	public function saveNBT(){
-		$this->namedtag->Items = new ListTag("Items", []);
+		$this->namedtag->Items->setValue([]);
 		$this->namedtag->Items->setTagType(NBT::TAG_Compound);
```

Comment:
I'm guessing this line is unnecessary because https://github.com/Muqsit/PocketMine-MP/blob/b7100070e2b2071a1b38579d337362e579dfbef9/src/pocketmine/tile/Chest.php#L51